Source/WebKit/UIProcess/API/Cocoa/_WKContextMenuElementInfo.mm

```diff
@@ -35,6 +41,28 @@ - (id)copyWithZone:(NSZone *)zone
     return [self retain];
 }
 
+- (void)dealloc
+{
+    if (WebCoreObjCScheduleDeallocateOnMainRunLoop(_WKContextMenuElementInfo.class, self))
+        return;
+    _contextMenuElementInfoMac->API::ContextMenuElementInfoMac::~ContextMenuElementInfoMac();
+    [super dealloc];
+}
+
+- (_WKHitTestResult *)hitTestResult
+{
+    auto& hitTestResultData = _contextMenuElementInfoMac->hitTestResultData();
+    auto apiHitTestResult = API::HitTestResult::create(hitTestResultData);
```

Comment:
This looks like nonatomic, readonly, copy to me instead of just nonatomic, readonly

Reply:
Fixed.